src/sentry/runner/commands/run.py

```diff
@@ -117,13 +111,12 @@ def smtp(bind, upgrade, noinput):
     'Number of child processes processing the queue. The '
     'default is the number of CPUs available on your '
     'system.'))
-@click.option('--loglevel', '-l', default='WARNING', help='Logging level.',
-    type=CaseInsensitiveChoice(CELERY_LOG_LEVELS))
 @click.option('--logfile', '-f', help=(
     'Path to log file. If no logfile is specified, stderr is used.'))
 @click.option('--quiet', '-q', is_flag=True, default=False)
 @click.option('--no-color', is_flag=True, default=False)
 @click.option('--autoreload', is_flag=True, default=False, help='Enable autoreloading.')
+@log_level_option
```

Comment:
This is useless because we tell celery that it's logger is configured elsewhere.